ProjFS.Linux/PrjFSLib.Linux.Managed/Interop/ProjFS.Constants.cs

```diff
@@ -9,11 +9,12 @@ internal static class Constants
             public const ulong PROJFS_ALLOW = 0x01;
             public const ulong PROJFS_DENY = 0x02;
             public const ulong PROJFS_CLOSE_WRITE = 0x00000008;
-            public const ulong PROJFS_OPEN = 0x00000020;
-            public const ulong PROJFS_DELETE_SELF = 0x00000400;
-            public const ulong PROJFS_MOVE_SELF = 0x00000800;
+            public const ulong PROJFS_MOVE = 0x000000C0;
```

Comment:
Why is `PROJFS_MOVE` `0x000000C0`?  (Wondering about the use of `C` vs a single bit)

Reply:
Well spotted!  Because Linux inotify delivers one or two messages for `rename()` (i.e., move) operations, [`IN_MOVE_FROM` and `IN_MOVE_TO`](https://github.com/torvalds/linux/blob/a4ccb5f9dc6c4fb4d4c0a9d73a911986f20ec88a/include/uapi/linux/inotify.h#L36), and then provides an [`IN_MOVE`](https://github.com/torvalds/linux/blob/a4ccb5f9dc6c4fb4d4c0a9d73a911986f20ec88a/include/uapi/linux/inotify.h#L50) convenience flag which combines the two.

The same is also true for fanotify [`FAN_MOVE`](https://github.com/torvalds/linux/blob/a4ccb5f9dc6c4fb4d4c0a9d73a911986f20ec88a/include/uapi/linux/fanotify.h#L34), because inotify and fanotify are aligned, and since we're trying to align libprojfs's events too -- in case we can potentially replace our FUSE libprofjs "shim" with some combination of OverlayFS and inotify/fanotify -- hence the matching definition here.